requirements.txt

```diff
@@ -0,0 +1 @@
+codespell==2.2.6
```

Comment:
As this is now using codespell from apt-get, we shouldn't need all the Python setup anymore?

Reply:
Removed